ComSemApp/templates/ComSemApp/teacher/course.html

```diff
@@ -93,10 +93,11 @@ <h4 class="m-t-0 header-title"><b>Worksheets</b></h4>
 				$(".release_worksheet_button").on("click", function(e){
 					$worksheet_row = $(this).parents('tr')
 					release_worksheet_url = $worksheet_row.attr('release_url')
-					cs_confirm("Release Worksheet", "Are you sure you'd like to release this worksheet?\nIf so, you will no longer be able to edit or delete it.", 'warning', "Yes, release it", release_worksheet);
+					cs_confirm("Release Worksheet", "Are you sure you'd like to release this worksheet?\nIf so, you will no longer be able to delete it.", 'warning', "Yes, release it", release_worksheet);
```

Comment:
Maybe this could describe the limited edit functionality for released worksheets:
```suggestion
					cs_confirm("Release Worksheet", "Are you sure you'd like to release this worksheet?\nIf so, you will no longer be able to delete it, create new expressions, or reassign or delete existing ones.", 'warning', "Yes, release it", release_worksheet);
```